yoti/src/YotiHelper.php

```diff
@@ -746,13 +747,14 @@ public static function getConfig() {
   /**
    * Get Yoti Dashboard app URL.
    *
-   * @deprecated use `yoti.sdk` service instead.
+   * @deprecated `yoti_app_id` will be removed in next major release.
```

Comment:
Do you think we should expand on this a bit more? When I first read it i didn't see how `yoti_app_id` was related, I think the `-` helps. Also was thinking we could say what is happening with login, so clients don't think they can't log in or something
```suggestion
   * @deprecated - `yoti_app_id` will be removed in next major release. Login will be through a Modal QR.
```

Reply:
I will remove the other `@deprecated` comment - I don't think it's required as there will be a single task to remove all occurrences of `yoti_app_id`